doc/content/guides/getting-started/console/_index.md

```diff
@@ -45,7 +44,9 @@ Devices are created within applications.
 
 ### Link application
 
-In order to send uplinks and receive downlinks from your device, you must link the Application Server to the Network Server. In order to do this, create an API key for the Application Server by going to **API keys** in the left menu of your application, and then clicking **+ Add API Key**.
+If you haven't unchecked the "Link automatically" checkbox during creation, your device will be automatically linked to the Application Server. You can skip this section in this case.
```

Comment:
"your device will be automatically linked to the Application Server"

I think there are some wrong words in there.